custom_endpoint/custom_endpoint.routing.yml

```diff
@@ -0,0 +1,8 @@
+custom_endpoint.jsonapi:
+  path: '/api/currentuser'
+  defaults:
+    _controller: '\Drupal\custom_endpoint\Controller\CustomEndpointController::renderApi'
+    _title: 'Current user data'
+  methods: ['GET']
+  requirements:
+    _access: 'TRUE'
```

Comment:
all your files should end with one empty line

Reply:
Done